--- configure.ac	2017-05-30 15:32:44.207217705 +0000
+++ configure.ac.orig	2017-05-30 15:32:01.244117123 +0000
@@ -29,13 +29,13 @@
 AC_ARG_WITH(cppunit,
         [  --without-cppunit       do not use CPPUNIT])

-#if test "$with_cppunit" = "no" ; then
+if test "$with_cppunit" = "no" ; then
    CPPUNIT_PATH="No_CPPUNIT"
    CPPUNIT_INCLUDE=
    CPPUNIT_LIBS=
-#else
-#   AM_PATH_CPPUNIT(1.10.2)
-#fi
+else
+   AM_PATH_CPPUNIT(1.10.2)
+fi

 if test "$CALLER" = "ANT" ; then
 CPPUNIT_CFLAGS="$CPPUNIT_CFLAGS -DZKSERVER_CMD=\"\\\"${base_dir}/src/c/tests/zkServer.sh\\\"\""
